app/scripts/modules/core/utils/renderIfFeature.directive.ts

```diff
@@ -0,0 +1,34 @@
+import {module} from 'angular';
+import {DirectiveFactory} from './tsDecorators/directiveFactoryDecorator';
+
+class RenderIfFeatureDirectiveController {
+  public $attrs: ng.IAttributes;
+  public $element: JQuery;
+
+  constructor(private settings: any) {}
+
+  public initialize(): void {
+    let feature: string = this.$attrs['renderIfFeature'];
+    if (feature && !this.settings.feature[feature]) {
+      this.$element.addClass('hidden');
```

Comment:
You might get into some weird priority stuff with the directive firing and initializing after the rest of the component, so it'll be a really fun thing for you to work through.